rust/lance-encoding/src/statistics.rs

```diff
@@ -0,0 +1,64 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: Copyright The Lance Authors
+
+use std::collections::HashMap;
+
+use arrow_array::ArrayRef;
+
+// A temporary and overly simplified sketch for arrow array statics,
+// The data structures and trait signiture here are totally temporary and of no use.
+// It is here only for filling in the `CoreArrayEncodingStrategy` field. 
+#[derive(Debug, Default)]
+pub struct Stats {
+    _stats_hashmap: HashMap<String, String>,
```

Comment:
Is this a hashmap because we expect new encodings to want to capture new statistics that we don't yet know about?

What entries do you envision this having?

Reply:
I am not sure about using a hashmap in the end, it's just a convenient way to put a hashmap as a placeholder,